beetmoverscript/docker.d/init_worker.sh

```diff
@@ -59,24 +58,41 @@ case $COT_PRODUCT in
         exit 1
         ;;
     esac
-    export TASKCLUSTER_SCOPE_PREFIX="project:comm:thunderbird:releng:${PROJECT_NAME}:"
     ;;
   mobile)
-    test_var_set 'MAVEN_ID'
-    test_var_set 'MAVEN_KEY'
-    test_var_set 'MAVEN_NIGHTLY_ID'
-    test_var_set 'MAVEN_NIGHTLY_KEY'
-    export TASKCLUSTER_SCOPE_PREFIX="project:mobile:android-components:releng:${PROJECT_NAME}:"
+    case $ENV in
+      dev|fake-prod)
+        test_var_set 'MAVEN_ID'
+        test_var_set 'MAVEN_KEY'
+        test_var_set 'MAVEN_NIGHTLY_ID'
+        test_var_set 'MAVEN_NIGHTLY_KEY'
+        test_var_set 'DEP_ID'
+        test_var_set 'DEP_KEY'
+        ;;
+      prod)
+        test_var_set 'MAVEN_ID'
+        test_var_set 'MAVEN_KEY'
+        test_var_set 'MAVEN_NIGHTLY_ID'
+        test_var_set 'MAVEN_NIGHTLY_KEY'
+        test_var_set 'DEP_ID'
```

Comment:
Note to self: I need to add these in SOPS + redeploy dev workers

Reply:
@CC @hneiva I think we'll have to make sure we have these setup properly in SOPS + cloudops-infra. I may have set them to dummy values last time, but if I didn't, we need to make sure there are valid values there.